web/src/pages/context/LastInputContext.tsx

```diff
@@ -0,0 +1,34 @@
+import { createContext, useContext, useState } from 'react'
+import { Multiflash } from '../../api/generated'
+
+const LastInputContext = createContext<
+  | { lastInput: Multiflash; setLastInput: (input: Multiflash) => void }
+  | undefined
+>(undefined)
```

Comment:
Why that name and these types? Currently, only temperature is used. IMO it would make more sense to remove pressure and composition from this context, and instead put the data from TResult into it. Perhaps rename LastInputContext to MainPageContext and TResults to TMainPage, and extend TMainPage to include temperature?

Reply:
I imaged we would use this context, along with a "resultContext", in future refactors.
So having the whole "lastInput" object gives it more potential for the future.